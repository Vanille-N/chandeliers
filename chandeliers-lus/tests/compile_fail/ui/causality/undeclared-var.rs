chandeliers_lus::decl! {
    node foo() returns (i: int);
    let i = absent; tel;
}

fn main() {}
