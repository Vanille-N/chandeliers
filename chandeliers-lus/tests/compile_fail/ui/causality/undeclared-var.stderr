error: Variable absent not found in the context.
 --> tests/compile_fail/ui/causality/undeclared-var.rs:3:13
  |
3 |     let i = absent; tel;
  |             ^^^^^^
  |
  = note: Perhaps you meant one of the local variables: i
  = note: or one of the global variables: (none declared)
